e2e/specs/film.spec.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { test, expect } from '@playwright/test';
+import { mockRoute } from '../apiMock';
+import { zendeskMock } from '../mockResponses';
+
+test.beforeEach(async ({ page }) => {
+  const filmFrontpage = mockRoute({
+    page,
+    path: '**/frontpage-api/v1/filmfrontpage/',
+    fixture: 'film_frontpage',
+  });
+
+  const allMovies = mockRoute({
+    page,
+    path: '**/search-api/v1/search/*',
+    fixture: 'film_all_movies',
+  });
+
+  const zendeskToken = mockRoute({
+    page,
+    path: '**/get_zendesk_token',
+    fixture: 'film_zendesk_token',
+    overrideValue: JSON.stringify(zendeskMock),
+  });
```

Comment:
Hvorfor ligger ikke denne mocken i `apiMocks`?

Reply:
Den gjør det? Fixture ligger der, du som la den til for 4 uker siden :+1: 
Ser ut som du skrev feil navn på zendesk_token så det ikke ble generert riktig for concept  🥇